api/rest/v2/observation.md

```diff
@@ -208,62 +316,72 @@ Response:
         },
         "geoId/06": {
           "orderedFacets": [
-            {
-              "facetId": "2176550201",
-              "observations": [
-                {
-                  "date": "2015",
-                  "value": 38904296
-                }
-              ]
-            },
-            ...
+            "earliestDate" : "2015",
+            "facetId" : "2176550201",
+            "latestDate" : "2015",
+            "obsCount" : 1,
+            "observations" : [
+              {
+                "date" : "2015",
+                "value" : 38904296
+              }
+            ]
+          },
+          ...
           ]
         }
       }
     }
   },
   "facets" {
-    "2176550201": {
-      "importName": "USCensusPEP_Annual_Population",
-      "provenanceUrl": "https://www2.census.gov/programs-surveys/popest/tables",
-      "measurementMethod": "CensusPEPSurvey",
-      "observationPeriod": "P1Y"
-    },
+      "2176550201" : {
+         "importName" : "USCensusPEP_Annual_Population",
+         "measurementMethod" : "CensusPEPSurvey",
+         "observationPeriod" : "P1Y",
+         "provenanceUrl" : "https://www2.census.gov/programs-surveys/popest/tables"
+      },
     ...
   }
+}
 ```
 {: .example-box-content .scroll}
 
-### Example 3: Latest observation for all California counties
+### Example 3: Get the latest observations for all California counties
 
-In this example, we use the [chained property
+In this example, we use the [chained expression
 (`+`)](/api/rest/v2/#relation-expressions) to specify "all contained places in
 [California](https://datacommons.org/browser/geoId/06) (dcid: `geoId/06`) of
 type `County`". Then we specify the select fields to request actual observations
 with date and value for each variable
-([Count_Person](https://datacommons.org/tools/statvar#sv=Count_Person)) and
+([`Count_Person`](https://datacommons.org/tools/statvar#sv=Count_Person)) and
 entity (all counties in California).
 
+Note: When sending a GET request, you need to use the following escape codes for reserved characters:
```

Comment:
should these notes go somewhere more prominent and in its own section that lists all the escape codes needed ..? because right now, reader has to look through each example to get the escape codes

Reply:
Yes, I was planning to add them to the "getting started" page, in another PR and remove these (and link to it instead). I agree it needs to be centralized. Let's leave it for now, and I will make sure you review that PR. (This is the problem with trying to do small, standalone PRs. :-))

Reply:
BTW, what is really weird is that sometimes they're needed and sometimes they're not. Like -> and <- seem to work in the examples for the node API, but not here. I don't quite understand why.

Reply:
doing in another PR makes sense to me! & that's interesting .. I'm not quite sure about that one